### DISCUSSION TITLE
Question about Page tag

### DISCUSSION THREAD

**Discussion by racine-p-a** [anansi-project/comicinfo#48] [Q&A]
Hello, I have a little question about the `<Page>` tag and its attributes. Is there some restrictions on the number of some types.

For example, could we have one day a usecase of a comic with multiple `FrontCover` or is it a complete non-sense ? Same question for all types.
- `FrontCover` : possible multiple ? Or always only one ?
- `InnerCover` : possible multiple ? Or always only one ?
- `Roundup` : possible multiple ? Or always only one ?
-  `Story` : well this one of course multiple pages...
- `Advertisement` : i suppose multiple pages
- `Editorial` : i suppose multiple pages
- `Letters` : i suppose multiple pages
- `Preview` : i suppose multiple pages
- `BackCover` : possible multiple ? Or always only one ?
- Other : i suppose multiple pages
- Delete : i suppose multiple pages

Is there some restrictions or is it at will ? I might not be the only one wanting to know this. If it's possible, can you add this information on [your documentation](https://anansi-project.github.io/fr/docs/comicinfo/documentation#pages--comicpageinfo)

Sorry for all my questions/issues, I am working on a personal tool to generate *ComicInfo.xml*. It's almost finished, I just want to be sure to not mess with this norm.

In case somebody would be interested in using my tool, it's [here](https://racine.pro/outils/comicinfo_generator). The interface is ugly and awful but it's enough for me.

**Comment by lordwelch** [comment]
It is common to mark alternate covers if the appear in the archive with FrontCover

**Comment by racine-p-a** [comment]
Thank you for your answer. So... multiple `FrontCover` is a possibility ?
And what about the other types ? No restriction at all on their quantity ?

**Reply by lordwelch** [reply]
No there is not a restriction
